src/ozone/ozone.py

```diff
@@ -91,7 +91,8 @@ def __init__(
 
         if self.file_name == "air_quality":
             warnings.warn(
-                "You have not specified a custom save file name. Existing files with the same name may be overwritten!"
+                "You have not specified a custom save file name. "
```

Comment:
Ahhh yes the cause of that annoying linter check fail

Reply:
Black is not able to fix that for us, apparently.